tests/lib/constants/messages_paths.py

```diff
@@ -75,6 +75,13 @@
     "pds_name_error_single_request_item_id"
 ]
 
+SUCCESSFUL_MESSAGE_IDS = [
+    "2aUxQER3co3kXdvTKrhbSFHRRxR",
```

Comment:
Is there any difference between these messages, just wondering if a Map would be more appropriate to give them individual Keys ?